src/main/scala/io/qbeast/context/QbeastContext.scala

```diff
@@ -65,7 +65,7 @@ object QbeastContext
 
   // Override methods from QbeastContext
 
-  override def config: Config = current.config
```

Comment:
Shouldn't it be `current.config`, as it was previously?

Reply:
It is a possibility! Have you tried with this modification?

Reply:
Nein, I just did it and the same error appears :cry:

Reply:
Change it to current.config in 888da87